addon/components/inputs/datetime.js

```diff
@@ -43,12 +53,14 @@ export default AbstractInput.extend({
   },
 
   @readOnly
-  @computed('defaultToCurrentDateTime', 'value')
-  currentValue (defaultToCurrentDateTime, value) {
-    if (!defaultToCurrentDateTime || value) {
-      return value
+  @computed('defaultToCurrentDateTime', 'timezone', 'storedDateTimeValue')
```

Comment:
Why are you moving away from `value`? If consumers decide to pass in a value directly into the `formValue`, this renderer won't update with the correct value. It only seems to initialize with the correct value at initialization, not updates.